src/loc/LocItem.tsx

```diff
@@ -240,51 +200,26 @@ export class FileItem extends LocItem {
     }
 }
 
-export class LinkItem extends LocItem {
-
-    constructor(commonData: CommonData, linkData?: LinkData) {
-        super(commonData);
-        this._linkData = linkData;
-    }
-
-    readonly _linkData?: LinkData;
+export class LinkItem extends AbstractLocItem<LinkData> {
 
     override title() {
-        return this._linkData?.nature || "-";
-    }
-
-    override hasData(): boolean {
-        return super.hasData() && this._linkData !== undefined;
-    }
-
-    metadataData(): MetadataData {
-        throw new Error("Method not implemented.");
+        return this.data()?.nature || "-";
```

Comment:
See above.

Reply:
Yse, indeed. `?` removed evrywhere it's useless.